smartgymapi/models/gym.py

```diff
@@ -2,14 +2,15 @@
 import uuid
 
 from sqlalchemy import Column, String
+
 from sqlalchemy_utils import UUIDType
 
-from smartgymapi.models.meta import Base, DBSession as session
+from smartgymapi.models.meta import Base, DBSession as session, LineageBase
 
 log = logging.getLogger(__name__)
 
 
-class Gym(Base):
+class Gym(Base, LineageBase):
```

Comment:
aangezien er geen gym factory en endpoint zijn is LineageBase (nog?) niet nodig.